### PR TITLE
Add LegendNation.GG

### PR DESCRIPTION
**Pull request by ld338**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x]  I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x]  I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x]  I have created the manifest.json with the required values

### CODE REVIEW

**Comment by ld338** [comment]
Its online, there was just a wrong record.
https://mcapi.us/server/status?ip=legendnation.gg

File: minecraft_servers/legendnation/manifest.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		"%.legendnation.gg"
+	],
+	"supported_languages": [
+		"de",
+		"en"
+	],
+	"social": {
+		"web": "https://www.legendnation.gg",
+		"web_shop": "https://legendnation.gg/shop",
+		"web_support": "https://legendnation.gg/support",
```
Comment (JakobBruening) [inline minecraft_servers/legendnation/manifest.json:15]: Page not available. Fix or remove.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please also fix json formatting. Use spaces as in our example manifest.json fiel.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please also fix json formatting. Use spaces as in our example manifest.json file.

**Comment by ld338** [comment]
Like this?

**Comment by ld338** [comment]
@JakobBruening

**Pull request merged**